package.json

```diff
@@ -60,7 +60,7 @@
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
     "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo src/icons -o dist/icons && shx cp -r src/popup/*.html dist/popup/",
-    "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content",
+    "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content && rollup -c --environment entry:lib/confirm-bar",
```

Comment:
I've never used rollup before, but not sure if we could use something like [**rollup-plugin-multi-entry**](https://github.com/rollup/rollup-plugin-multi-entry) to specify an array of entry points instead of calling the full `rollup -c --environment entry:*` command 3+ times.